section_to_course/__init__.py

```diff
@@ -2,4 +2,4 @@
 Factors sections from Open edX courses into their own new course.
 """
 
-__version__ = '0.2.0'
+__version__ = '0.3.0'
```

Comment:
```suggestion
__version__ = '0.2.1'
```

This version doesn't add any new features or change any APIs, but fixes a build.

Reply:
This is the content of the Python package:
```
section_to_course
  ├── __init__.py
  ├── admin.py
  ├── api
  │  ├── __init__.py
  │  ├── urls.py
  │  └── views.py
  ├── apps.py
  ├── compat.py
  ├── management
  │  ├── __init__.py
  │  └── commands
  │     ├── __init__.py
  │     └── section_to_course.py
  ├── migrations
  │  ├── 0001_initial.py
  │  └── __init__.py
  ├── models.py
  ├── settings
  │  ├── __init__.py
  │  └── common.py
  ├── static
  │  └── section_to_course
  │     └── js
  │        └── admin-tools.js
```

As long as we're not changing anything from there (or adding new files that would be included in the package), it does not make sense to create a new release - there will be no changes for people installing this. You can keep the version as-is and add the point from the CHANGELOG under the `Unreleased` section.

Reply:
Reverted the version changes and updated CHANGELOG `Unreleased` section.